src/api.rs

```diff
@@ -82,7 +82,7 @@ pub fn get_api_handler(collection: Arc<Collection>) -> Mount {
 }
 
 fn path_from_request(request: &Request) -> Result<PathBuf> {
-	let path_string = request.url.path().join("\\");
+	let path_string = request.url.path().join("/");
```

Comment:
I think this would break on Windows instead because the DB stores path using backslashes on that platform. Can you change this to using path::MAIN_SEPARATOR instead?